### ISSUE TITLE
[BUG]: Global CSS cannot be imported from within node_modules.

### ISSUE THREAD

**Issue by PJijin** [KRRISH96/render-smooth-image-react#1]
**Describe the bug**
I have just installed on nextjs and received this error.

```
./node_modules/render-smooth-image-react/build/style.css
Global CSS cannot be imported from within node_modules.
Read more: https://err.sh/next.js/css-npm
Location: node_modules/render-smooth-image-react/build/index.js
```

**To Reproduce**
Steps to reproduce the behaviour:
1. Set up nextjs project
2. Add this package
3. Insert code to smooth load image.

**Comment by KRRISH96** [comment]
@PJijin Thanks for reporting.

I am sorry for the delayed response, It might have slipped through the notifications.

I will look into this and publish a new version over this weekend.

**Comment by KRRISH96** [comment]
Fixed in `v0.0.9`

Closing issue!
